src/script/sign.cpp

```diff
@@ -570,14 +593,18 @@ class DummySignatureCreator final : public BaseSignatureCreator {
     bool CreateSchnorrSig(const SigningProvider& provider, std::vector<unsigned char>& sig, const XOnlyPubKey& pubkey, const uint256* leaf_hash, const uint256* tweak, SigVersion sigversion) const override
     {
         sig.assign(64, '\000');
+        if (m_include_sighash) {
+            sig.push_back((unsigned char)SIGHASH_ALL);
+        }
         return true;
     }
+    bool IsDummy() const override { return true; }
```

Comment:
Adding a method to ask an interface if it's a certain class seems like a design inversion to me. If we can't move the actual responsibility into the class, is there some more semantic name that we could consider? I see it's assigned to `use_largest`, so maybe `UseLargest`? (then further describe in the doc-comment  `Choose the largest stack size for worst case size estimation when using this signer.`).

Reply:
Changed to `UseLargest`